virtual_rainforest/models/soil/carbon.py

```diff
@@ -1,34 +1,60 @@
 """The ``models.soil.carbon`` module  simulates the soil carbon cycle for the Virtual
-Rainforest. At the moment four pools are modelled, these are low molecular weight carbon
-(LMWC), mineral associated organic matter (MAOM), microbial biomass, and particulate
-organic matter (POM).
+Rainforest. At the moment five pools are modelled, these are low molecular weight carbon
+(LMWC), mineral associated organic matter (MAOM), microbial biomass, particulate organic
+matter (POM), and POM degrading enzymes.
 """  # noqa: D205, D415
 
+from dataclasses import dataclass
+
 import numpy as np
 from numpy.typing import NDArray
-from scipy.constants import convert_temperature, gas_constant
 
-from virtual_rainforest.core.logger import LOGGER
+from virtual_rainforest.core.constants import CoreConsts
 from virtual_rainforest.models.soil.constants import SoilConsts
+from virtual_rainforest.models.soil.env_factors import (
+    calculate_environmental_factors,
```

Comment:
The name of the function feels a bit potentially misleading, it's fine for now but we might want to state more explicitly that is is about the effects, not the environment itself

Reply:
effect sounds good, originally called this `environmental_impact_factors` but felt like that sounded too much like it was impacts **on** the environment